project/Dependencies.scala

```diff
@@ -49,5 +40,5 @@ object Dependencies {
 
   // provide compatibility between scala 2.12 and 2.13
   // see https://github.com/scala/scala-collection-compat/issues/208
-  val scalaCollectionCompatDependencies = Seq("org.scala-lang.modules" %% "scala-collection-compat" % "2.12.0")
+  val scalaCollectionCompat = "org.scala-lang.modules" %% "scala-collection-compat" % "2.12.0"
```

Comment:
do we still need this dependency, given that 2.12 support is removed in this PR?

Reply:
Good spot! Done with 2b1a114!